### DISCUSSION TITLE
Radio buttons

### DISCUSSION THREAD

**Discussion by PaulBernhardt** [community/community#5313] [Projects and Issues]
I would like to be able to have a group of checkboxes where you have to pick one (and only one) of them. Something like:

Requires Security Review:
[ ] Yes
[ ] No

In particular, the user should have to explicitly opt in to one or the other and not rely on a default unchecked checkbox, and you should be able enforce that one of those boxes must be checked before the PR is merged. At the very least you should be able to opt it out of the task list system so you can still say all tasks need to be checked.

**Comment by gene1wood** [comment]
This was requested in #4318 a few months before this topic.
